OTRGui/src/game/game.cpp

```diff
@@ -67,7 +67,7 @@ void OTRGame::init(){
 		mat.shader = shader;
 	}
 
-	if(fs::exists("soh.exe") && !fs::exists("oot.otr")) {
+	if(fs::exists("soh.exe") || fs::exists("soh.elf") && !fs::exists("oot.otr")) {
```

Comment:
this won't work because `if(true || false && false)` returns `true`, meaning if we have `soh.exe` it doesn't matter if we have `oot.otr` or not

we need extra parens here, `if((true || false) && false)` returns `false`

so the line should be

```
if((fs::exists("soh.exe") || fs::exists("soh.elf")) && !fs::exists("oot.otr")) {
```

Reply:
Didn't work for me in testing. Still required I create a dummy `soh.exe` or it would fail, and second button would appear. `touch soh.exe`.  Please test solution.

Reply:
when you were testing did you test having a dummy `soh.exe` but also having a `oot.otr`?

Reply:
a simplified version
```                                                                                                 
#include <stdbool.h>
#include <stdio.h>

int main() {
if(true || false && false) {
 printf("without parens");
}

if((true || false) && false) {
 printf("with parens");
}

return 0;
}
```